views/js/controller/items/edit.js

```diff
@@ -37,12 +37,20 @@ function($, __, module, actions, lock, section){
         start() {
             const config = module.config();
 
-            const previewAction = actions.getBy('item-preview');
-            const authoringAction = actions.getBy('item-authoring');
-
-            if(previewAction){
-                previewAction.state.disabled = !config.isPreviewEnabled;
+            const getPreviewId = idx => `item-preview${idx ? `-${idx}` : ''}`;
+            const previewActions = []
+            for (let i = 0; i < 10; i++) {
```

Comment:
Why static `10` here? Can this number change?

Reply:
It is an arbitrary value. I've updated with a clearer value